### ISSUE TITLE
Options "key" and "keys" are not taken from instanceOptions

### ISSUE THREAD

**Issue by dumptyd** [farzher/fuzzysort#59]
When creating an instance with default options, the library doesn't make use of the `key` or `keys` options.

Example

```js
const fz = fuzzysort.new({
  key: 'someKey',
  // other options
});

fz.go('a', [{ someKey: 'a' }, { someKey: 'b' }]); // error

fz.go('a', [{ someKey: 'a' }, { someKey: 'b' }], { key: 'someKey' }); // works

**Comment by farzher** [comment]
yup... oops. to be honest i'd rather remove this weird feature than fix it. but that's a breaking change :(

**Comment by farzher** [comment]
"fixed" in v2.0.0
